api/envoy/extensions/filters/http/oauth2/v3alpha/oauth.proto

```diff
@@ -22,6 +22,21 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 //
 
 message OAuth2Credentials {
+  message CookieNames {
+    // Cookie name to hold OAuth bearer token value. When the authentication server validates the
+    // client and returns an authorization token back to the OAuth filter, no matter what format
+    // that token is, if :ref:`forward_bearer_token <envoy_v3_api_field_extensions.filters.http.oauth2.v3alpha.OAuth2Config.forward_bearer_token>`
+    // is set to true the filter will send over the bearer token as a cookie with this name to the
+    // upstream. Defaults to ``BearerToken``.
+    string bearer_token = 1;
+
+    // Cookie name to hold OAuth HMAC value. Defaults to ``OauthHMAC``.
+    string oauth_hmac = 2;
+
+    // Cookie name to hold OAuth expiry value. Defaults to ``OauthExpires``.
+    string oauth_expires = 3;
```

Comment:
Does this need one of the valid header value validation annotations? I'm not sure what the rules are for cookies? cc @asraa @yanavlasov

Reply:
Seems like this needs to be stricter than `well_known.HTTP_HEADER_VALUE`, https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#attributes.

Reply:
I would at minimum set that validation but maybe we also need a stricter one and/or do it in code? @asraa @yanavlasov @alyssawilk wdyt?

/wait

Reply:
OK. Probably adding: `HTTP_COOKIE_NAME` and `HTTP_COOKIE_VALUE` patterns to `well_known` in PGV will be useful. I'll try to have those regex patterns and PR it.